fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/sfcc/utils.rs

```diff
@@ -0,0 +1,133 @@
+use uom::{
+    si::{angle::degree, f64::*},
+    ConstZero,
+};
+
+pub struct SlatFlapControlComputerMisc {}
```

Comment:
```suggestion
pub(super) struct SlatFlapControlComputerMisc {}
```
As well for the member functions.